onnxscript/evaluator.py

```diff
@@ -361,6 +375,18 @@ def _eval(self, schema, inputs, attributes, closure):
         return _call_ort(schema, inputs, attributes, closure)
 
 
+class OnnxReferenceRuntimeEvaluator(Evaluator):
```

Comment:
Where is it used or tested?

Reply:
Good catch - I haven't done it yet

Reply:
Enabled tests for all atenlib functions and eager_test